package.json

```diff
@@ -44,6 +44,8 @@
     "@types/node": "^17.0.35",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.9",
+    "@types/testing-library__jest-dom": "^5.14.3",
+    "@types/testing-library__react": "^10.2.0",
```

Comment:
This shouldn't be needed if you rebase your PR?

Reply:
Already rebased?